.circleci/config.yml

```diff
@@ -74,6 +74,12 @@ mysql-container: &mysql-container
     MYSQL_ROOT_PASSWORD: ''
     MYSQL_DATABASE: circleci
 
+postgres-container: &postgres-container
+  image: circleci/postgres:10.5-alpine
+  environment:
+    POSTGRES_DB: circleci
+    POSTGRES_USER: root
```

Comment:
I think by default it should be `postgres`

Reply:
It was. We changed because of https://discuss.circleci.com/t/circleci-postgres-image-suddenly-causing-problems-role-postgres-not-found/25044